src/centos/7/Dockerfile

```diff
@@ -2,7 +2,11 @@ FROM centos:7
 
 # Install dependencies
 
-RUN yum install -y \
+RUN \
+    # Since the base image isn't actively maintained, force an upgrade of existing packages
+    yum update -y \
+    && \
+    yum install -y \
```

Comment:
This will make the dependency layer really large since the image hasn't been updated in a long time. What is the layer size difference caused by this change, and if it's big (relative to the size of the entire image), will you consider squashing this into a single layer with the base image e.g. https://stackoverflow.com/a/77052183/13771378